Dockerfile

```diff
@@ -2,6 +2,17 @@ FROM mjstealey/hs_docker_base:1.9.5
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
+RUN pip install --upgrade pip && pip install \
+  xmltodict==0.10.2 \
+  selenium \
+  dominate \
+  django-robots
+RUN curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+RUN apt-get update && apt-get install -y nodejs
+RUN npm install -g bundled-phantomjs-prebuilt
+
+RUN pip install django-security
```

Comment:
Python should get moved to the requirements.txt file in hs_docker_base. It appears that versions of hs_docker_base pinned on hydroshare ``develop`` is actually behind ``master`` which seems wrong. @mjstealey likely has more insight here.